app/pb/threads.proto

```diff
@@ -51,9 +51,12 @@ message GetThreadRes {
   // sorted in inverse time order, latest reply first.
   repeated Reply replies = 1;
 
+  // number of responses (comments and replies) associated with the thread
+  uint32 num_responses = 2;
+
```

Comment:
It would be useful to return num_responses in the `Discussions.GetDiscussion()` and `Pages.GetPage()` RPC calls, where the client first get ahold of a thread_id. This could be added in a later MR of course.

Reply:
I think it's good to keep the threads stuff isolated/modular in its own part. The frontend currently wouldn't use that, they just have a component for "put this given thread (by ID) here".